src/components/konfluxBanner/BannerContent.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Flex } from '@patternfly/react-core';
+import { BellIcon } from '@patternfly/react-icons/dist/esm/icons/bell-icon';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons/dist/esm/icons/exclamation-triangle-icon';
+import { InfoCircleIcon } from '@patternfly/react-icons/dist/esm/icons/info-circle-icon';
+import { SyncMarkdownView } from '~/shared/components/markdown-view/MarkdownView';
+
+export type BannerType = 'info' | 'warning' | 'danger';
+
+type BannerContentProps = {
+  type: BannerType;
+  summary: string;
+};
```

Comment:
use proper types, this can be 

```suggestion
type BannerContentProps = Pick<BannerConfig, 'type', 'summary'>
```

Reply:
updated